tigerpath/majors_and_certificates/majors/SOC_2018.json

```diff
@@ -36,37 +36,62 @@
       "name": "Departmental Courses",
       "max_counted": 1,
       "min_needed": "ALL",
-      "explanation": "Students are required to take a minimum of nine courses in sociology, including an upper limit of two cognate courses in other departments which must receive approval from Sociology in order to count toward the required nine. A cognate course is a Princeton class offered by another department that has substantial sociological content. All departmental courses or approved cognates that count toward the required nine must be taken for a grade and cannot be taken pass/D/fail. SOC 101, SOC 300, POL 345/SOC 305, and SOC 302 are requirements for the major. Collectively they are designed to help students carry out their junior and senior independent work. These courses expose students to the nature of sociological problems and theory, the logic of inquiry, the techniques of empirical investigation, and the elements of statistics. SOC 300 and POL 345/SOC 305 are usually taken in the fall of the junior year and are offered at that time to facilitate students who wish to study abroad in the spring. SOC 302 is normally offered in the spring.",
+      "explanation": "Sociology majors must take nine courses within the department. Up to two courses can be courses with sociological content that come from other departments. (The Director of Undergraduate Studies has to approve those courses after seeing a syllabus.) Within the nine sociology courses, the following four must be taken: SOC 101: Introduction to Sociology (taught in the fall and the spring and can be taken anytime, even after taking mid- and upper-level sociology courses), SOC 300: Claims and Evidence (taken in the Fall of the junior year to prepare students for the Junior Paper), SOC 301: Statistical Methods in Sociology, SOC 302: Sociological Theory.",
       "req_list": [
         {
           "name": "Required",
           "max_counted": 1,
           "min_needed": "ALL",
-          "explanation": "",
+          "explanation": "Within the nine sociology courses, the following four must be taken: SOC 101: Introduction to Sociology (taught in the fall and the spring and can be taken anytime, even after taking mid- and upper-level sociology courses), SOC 300: Claims and Evidence (taken in the Fall of the junior year to prepare students for the Junior Paper), SOC 301: Statistical Methods in Sociology, SOC 302: Sociological Theory.",
+          "completed_by_semester": 6,
           "course_list": [
             "SOC 101",
             "SOC 300",
-            "POL 345/SOC 305",
+            "SOC 301",

```

Comment:
Good update.
I think the 3rd point [here](https://sociology.princeton.edu/undergraduate-program/prospective-majors/faqs) might still apply, though, which seems to imply that now, instead of SOC 301, they can still take POL 345/SOC 305, or ORF 245/EGR 245, PSY 251, or WWS 200.

So maybe it would be best to split this course out into a separate req which allows any one of those courses to satisfy it, with that FAQ answer as the explanation.

Reply:
Ah indeed, [this page](https://sociology.princeton.edu/undergraduate-program/sophomores) confirms it, and is probably a better source, since it seems to be more up to date.